deliverables/README.md

```diff
@@ -1,14 +1,14 @@
 ## Deliverables
 #### Required
 ##### Questions
-- What is a framework?
-- What's the difference between a programming language and a framework?
-- What’s the difference between a library and a framework?
-- What languages have frameworks?
+- What is a framework? A framework is a collection of tools used as batched and recycled code which provide organization, a software library extended from a programming language.
+- What's the difference between a programming language and a framework? A framework directly works with things from a programming language in order to not repeat them again or have them interact with 
+- What’s the difference between a library and a framework? In a library, the code is called upon, the code is made to fit the library, while in a framework the code is directly plugged in and the framework is made to fit the code.
```

Comment:
You've got the distinction correct, but it's vice-versa from the way you have it.  

A library is meant to be embedded inside the code that you write.  Conversely the code you write is designed to be embedded within a framework (think picture frame).